### PR TITLE
Refactor/cleanup renaming

### PR DESCRIPTION
**Pull request by ghost**
## Description

* Moved apps under ui
* Moved plugins under apps
* Moved feed widget under ui/lib

## Checklist

- [ ] I have read the **README** document
- [ ] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [ ] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: ui/apps/auth-app/webpack.config.js [hunk 1]
```diff
@@ -7,6 +7,6 @@ module.exports = Object.assign(baseConfig, {
   context: path.resolve(__dirname),
   output: Object.assign(baseConfig.output, {
     path: path.resolve(__dirname, 'dist'),
-    publicPath: '/plugins/search/',
+    publicPath: '/apps/auth-app/',
```
Comment (kenshyx) [inline ui/apps/auth-app/webpack.config.js:10]: I think this one should target `search-app`?

File: ui/apps/notifications/webpack.config.js [hunk 2]
```diff
@@ -7,6 +7,6 @@ module.exports = Object.assign(baseConfig, {
   context: path.resolve(__dirname),
   output: Object.assign(baseConfig.output, {
     path: path.resolve(__dirname, 'dist'),
-    publicPath: '/plugins/app-center/',
+    publicPath: '/apps/notifications/',
```
Comment (kenshyx) [inline ui/apps/notifications/webpack.config.js:10]: this should be `app-center`

File: ui/apps/akasha/webpack.config.js [hunk 3]
```diff
@@ -7,6 +7,6 @@ module.exports = Object.assign(baseConfig, {
   context: path.resolve(__dirname),
   output: Object.assign(baseConfig.output, {
     path: path.resolve(__dirname, 'dist'),
-    publicPath: '/plugins/legal/',
+    publicPath: '/apps/akasha/',
```
Comment (kenshyx) [inline ui/apps/akasha/webpack.config.js:10]: this should be `legal`

File: ui/apps/auth-app/webpack.config.js (hunk 1 above)
Comment (ghost) [reply ui/apps/auth-app/webpack.config.js:10]: Think this is correct, it is inside the auth-app folder? Github is showing the change weird though.

**Review by kenshyx — APPROVED** [review]

**Pull request merged**